x-pack/test/detection_engine_api_integration/security_and_spaces/tests/generating_signals.ts

```diff
@@ -70,6 +70,19 @@ export default ({ getService }: FtrProviderContext) => {
         expect(signalsOpen.hits.hits.length).greaterThan(0);
       });
 
+      it('should abide by max_signals > 100', async () => {
+        const maxSignals = 500;
+        const rule: QueryCreateSchema = {
+          ...getRuleForSignalTesting(['auditbeat-*']),
+          max_signals: maxSignals,
+        };
+        const { id } = await createRule(supertest, rule);
+        await waitForRuleSuccessOrStatus(supertest, id);
+        await waitForSignalsToBePresent(supertest, maxSignals, [id]);
+        const signalsOpen = await getSignalsByIds(supertest, [id], maxSignals);
```

Comment:
```ts
const signalsOpen = await getSignalsByIds(supertest, [id], maxSignals);
```

Is this because the limit is 10 by default as that's the default Elastic search limit?

Reply:
bingo